modules/api-resources/api-resources-full/src/main/webapp/WEB-INF/web.xml

```diff
@@ -24,108 +24,359 @@
+                org.wso2.carbon.identity.api.dispatcher.core.JsonProcessingExceptionMapper,
+                org.wso2.carbon.identity.api.dispatcher.core.APIErrorExceptionMapper,
+                org.wso2.carbon.identity.api.dispatcher.core.InputValidationExceptionMapper,
+                org.wso2.carbon.identity.api.dispatcher.core.DefaultExceptionMapper,
+                org.apache.cxf.jaxrs.ext.search.SearchContextProvider
             </param-value>
         </init-param>
     </servlet>
     <servlet-mapping>
-        <servlet-name>IdentityRecoveryV0_9ApiServlet</servlet-name>
-        <url-pattern>/identity/recovery/v0.9/*</url-pattern>
+        <servlet-name>OrganizationServerV2Servlet</servlet-name>
+        <url-pattern>/o/server/v2/*</url-pattern>
     </servlet-mapping>
 
-    <!--  Servlet instance for /identity/consent-mgt/v1.0 -->
+    <!--  Servlet instance for /users/v1/me/organizations -->
     <servlet>
-        <servlet-name>IdentityConsentMgtV1ApiServlet</servlet-name>
+        <servlet-name>OrgUserServlet</servlet-name>
         <servlet-class>
             org.apache.cxf.jaxrs.servlet.CXFNonSpringJaxrsServlet
         </servlet-class>
         <load-on-startup>1</load-on-startup>
         <init-param>
             <param-name>jaxrs.serviceClasses</param-name>
             <param-value>
-                org.wso2.carbon.consent.mgt.endpoint.ConsentsApi
+                org.wso2.carbon.identity.rest.api.user.organization.v1.RootApi,
+                org.wso2.carbon.identity.rest.api.user.organization.v1.MeApi
             </param-value>
         </init-param>
         <init-param>
             <param-name>jaxrs.providers</param-name>
             <param-value>
-                com.fasterxml.jackson.jaxrs.json.JacksonJsonProvider,
-                org.wso2.carbon.consent.mgt.endpoint.expmapper.ConsentEndpointExceptionMapper,
+                org.wso2.carbon.identity.servlet.mgt.CustomJacksonJsonProvider,
+                org.wso2.carbon.identity.api.dispatcher.core.JsonProcessingExceptionMapper,
+                org.wso2.carbon.identity.api.dispatcher.core.APIErrorExceptionMapper,
+                org.wso2.carbon.identity.api.dispatcher.core.InputValidationExceptionMapper,
+                org.wso2.carbon.identity.api.dispatcher.core.DefaultExceptionMapper
+            </param-value>
+        </init-param>
+        <init-param>
+            <param-name>jaxrs.inInterceptors</param-name>
+            <param-value>
+                org.wso2.carbon.identity.servlet.mgt.CustomBeanValidationInterceptor
             </param-value>
         </init-param>
     </servlet>
     <servlet-mapping>
-        <servlet-name>IdentityConsentMgtV1ApiServlet</servlet-name>
-        <url-pattern>/identity/consent-mgt/v1.0/*</url-pattern>
+        <servlet-name>OrgUserServlet</servlet-name>
+        <url-pattern>/users/v1/me/organizations/*</url-pattern>
```

Comment:
This won't be mapping `users/v1/me/organizations` API to this servlet. Better to have `/users/v1/me/organizations*` as the pattern